src/components/IconCustomizer.svelte

```
@@ -1,63 +1,46 @@
+				if (isContentTypeSvg(contentType)) {
+					customizedIcon.selectSvgIcon(iconContent);
+				} else {
+					_.flow(IconService.mkIconUrl, customizedIcon.selectImageIcon)($customizedIcon.iconId);
+				}
+			})
+			.catch((error) => {
+				throw new Error('Error fetching icon', error);
+			})
+			.finally(() => {
+				previousIconId = $customizedIcon.iconId;
+			});
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)